kytos/utils/config.py

```diff
@@ -140,19 +141,23 @@ def get_remote_metadata(cls):
         """Return kytos metadata."""
         kytos_api = KytosConfig().config.get('kytos', 'api')
         meta_uri = kytos_api + 'api/kytos/core/metadata/'
-        meta_file = urllib.request.urlopen(meta_uri).read()
+        meta_file = urlopen(meta_uri).read()
         metadata = json.loads(meta_file)
         return metadata
 
     @classmethod
     def check_versions(cls):
         """Check if kytos and kytos-utils metadata are compatible."""
-        kytos_metadata = cls.get_remote_metadata()
-        kutils_metadata = cls.get_metadata()
-        kytos_version = kytos_metadata.get('__version__')
-        kutils_version = kutils_metadata.get('__version__')
-
-        if kytos_version != kutils_version:
-            logger = logging.getLogger()
-            logger.warning('kytos (%s) and kytos utils (%s) versions '
-                           'are not equal.', kytos_version, kutils_version)
+        try:
+            kytos_metadata = cls.get_remote_metadata()
+            kytos_version = kytos_metadata.get('__version__')
+        except URLError:
+            LOG.debug('kytos is not running.')
```

Comment:
It's better to not assume that kytos is not running: it may be a connection problem. So 'Couldn't connect to kytos server.' is more precise. Also, please include the original error message from the exception object.